polaris-react/src/components/Checkbox/Checkbox.tsx

```diff
@@ -37,16 +36,30 @@ export interface CheckboxProps {
   name?: string;
   /** Value for form input */
   value?: string;
-  /** Display an error message */
-  error?: Error | boolean;
   /** Callback when checkbox is toggled */
   onChange?(newChecked: boolean, id: string): void;
   /** Callback when checkbox is focused */
   onFocus?(): void;
   /** Callback when focus is removed */
   onBlur?(): void;
+  /** Added to the wrapping label */
+  labelClassName?: string;
+  /** Grow to fill the space. Equivalent to width: 100%; height: 100% */
+  fill?: boolean;
```

Comment:
Should this be `ResponsiveProp<boolean>` or are we being opinionated in particular circumstances? I suppose this is likely fine though for now, unless a use case sprouts up.

Reply:
🤦 Forgot to copy+paste this over... Good catch - I've brought that over 👍